website/pages/api-docs/events.mdx

```diff
@@ -0,0 +1,626 @@
+---
+layout: api
+page_title: Events - HTTP API
+sidebar_title: Events
+description: |-
+  The /event endpoints interact with the local Nomad agent to interact with
+  members and servers.
+---
+
+# Events HTTP API
+
+The `/event` endpoints are used to query for recent events.
+
+## Event Stream
+
+This endpoint streams a servers backlog off events as we as new events as they occur.
+The stream will be kept alive until the connection is closed.
+
+| Method | Path               | Produces           |
+| ------ | ------------------ | ------------------ |
+| `GET`  | `/v1/event/stream` | `application/json` |
+
+
+The Table below shows this endpoint's [required ACLs](/api-docs#acls). Due to
+the nature of this endpoint different topics require different policies.
+
+| Topic        | ACL Required         |
+| ------------ | -------------------- |
+| `*`          | `management`         |
+| `Job`        | `namespace:read-job` |
+| `Alloc`      | `namespace:read-job` |
+| `Deployment` | `namespace:read-job` |
+| `Eval`       | `namespace:read-job` |
+| `Node`       | `node:read`          |
+
+### Parameters
+
+- `index` `(int: 0)` - Specifies the index to start streaming events from. If
+  the requeste index is no longer in the buffer the stream will start at the
+  nearest index.
+
+- `topic` `(topic:filter_key: "*:*")` - Specifies a topic to subscribe to and
+  filter on. By default all topics are subscribed to. Multiple topics should be
+  specified by passing multiple `topic` parameters. A valid topic parameter
+  includes a `topic` type and an optional `filter_key` separated by a colon
+  `:`. As an example `?topic=Deployment:redis` would subscribe to all
+  `Deployment` events for a job redis. To only subscribe to `Node` events a
```

Comment:
Can there be multiple filters per topic?

Reply:
yes will add to this to reflect